packages/fxa-auth-server/docs/swagger/shared/descriptions.ts

```diff
@@ -112,8 +145,29 @@ const DESCRIPTIONS = {
   queryKeys:
     'Indicates whether a new `keyFetchToken` is required, default to `false`.',
   payload: 'Opaque payload to be forwarded to the device.',
+  paymentMethodId: 'A unique identifier for the payment method.',
+  paymentProvider:
+    'The third-party that assists in the electronic payments (e.g. PayPal, Stripe).',
```

Comment:
We often refer to these as payment processors. Could use that term in here for consistency.